cosmwasm/utils.js

```diff
@@ -36,6 +36,32 @@ const { normalizeBech32 } = require('@cosmjs/encoding');
 const DEFAULT_MAX_UINT_BITS_EVM = 256;
 const DEFAULT_MAX_DECIMALS_WHEN_TRUNCATING_EVM = 255;
 
+const CONTRACT_NAME_COORDINATOR = 'Coordinator';
```

Comment:
Instead of all these globals, define an object that maps the contract name to an object containing the corresponding instantiate fn and the scope type. Standardize the arguments to all instantiate fn so they can be dispatched generically. We do something analogous for sui/deploy-contract.js for contract specific code